javascript/javascript1/week2/candy-helper.js

```diff
@@ -0,0 +1,62 @@
+let boughtCandyPrices = [];
+
+function addCandy(candyType , weight) {
+  if(candyType === 'Sweet')
```

Comment:
The commented out solution is way better, just use that one. In general if you end up writing very long if-else statement, you should look for a better way of doing it, cause those are hard to read and to understand :) it also might be a good idea to do `candyType.toLowercase()` and compare with lower case candy names. But that is not a mistake by any means and it fits the requerements from the assignment.